superbench/benchmarks/micro_benchmarks/directx_mem_bw_performance/BenchmarkOptions.h

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "../directx_utils/Options.h"
+#include "GPUMemRwBw.h"
+
+enum Memtype {
+    Read,
+    Write,
+    ReadWrite,
+};
+
+struct UInt3 {
+    unsigned int x;
+    unsigned int y;
+    unsigned int z;
+};
+
+class BenchmarkOptions : public Options {
+  protected:
+    UInt3 get_cmd_line_argument_uint3(const std::string &option, UInt3 defaults) {
```

Comment:
>UInt3 [](http://example.com/codeflow?start=65&length=5)

const UINT3&

Reply:
revised, thanks

Reply:
well, you can return a UInt3 but use the UInt3 reference in the function's parameters.

Reply:
ok, updated to and move it to Options.h
```
UInt3 get_cmd_line_argument_uint3(const std::string &option, const UInt3 &defaults)
```